packages/web5/test/dids/did_web_test.dart

```diff
@@ -77,13 +78,14 @@ void main() {
   });
 
   group('DidWeb', () {
-    test('should return invalid did with bad data', () async {
-      final result = await DidWeb.resolve('bogus');
-      expect(result, DidResolutionResult.invalidDid());
-    });
+    // test('should return invalid did with bad data', () async {
+    //   final result = await DidWeb.resolve('bogus');
+    //   expect(result, DidResolutionResult.invalidDid());
+    // });
```

Comment:
Are these gonna get added back in later?

Reply:
this test is are no longer applicable for method specific resolvers because the method signature changed to expecting Did vs just a string. the global resolver takes a string. calling `Did.parse('bogus');` would fail